An error occurred while executing the following cell:
------------------
"""
Author : Ming Liang Ang
Based on : https://github.com/probml/pmtk3/blob/master/demos/gibbsDemoIsing.m
"""

import numpy as np
import matplotlib.pyplot as plt
import jax
import jax.numpy as jnp

try:
    import probml_utils as pml
except ModuleNotFoundError:
    %pip install -qq git+https://github.com/probml/probml-utils.git
    import probml_utils as pml
------------------

---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
/tmp/ipykernel_4163/3115004030.py in <module>
     11 try:
---> 12     import probml_utils as pml
     13 except ModuleNotFoundError:

ModuleNotFoundError: No module named 'probml_utils'

During handling of the above exception, another exception occurred:

ModuleNotFoundError                       Traceback (most recent call last)
/tmp/ipykernel_4163/3115004030.py in <module>
     13 except ModuleNotFoundError:
     14     get_ipython().run_line_magic('pip', 'install -qq git+https://github.com/probml/probml-utils.git')
---> 15     import probml_utils as pml

ModuleNotFoundError: No module named 'probml_utils'
ModuleNotFoundError: No module named 'probml_utils'
